tests/ProjectCodeTest.php

```diff
@@ -116,11 +116,33 @@ public function testHasThrowsAnnotation($method)
     public function testHasCorrespondingStaticAnalysisFile($method)
     {
         $doc = $method->getDocComment();
+        $methodName = $method->getName();
+
         if($doc === false || strpos($doc, '@psalm-assert') === false) {
             $this->addToAssertionCount(1);
             return;
         }
 
+        if (\in_array(
+            $methodName,
+            array(
+                'alnum',
+                'digits',
+                'upper',
+                'lower',
+                'readable',
+                'writable',
+                'uuid',
+            ),
+            true
+        )) {
+            self::markTestIncomplete(\sprintf(
+                'Could not design assertions for "%s" due to missing !empty test scenario'
+                . "\n\nSee https://github.com/vimeo/psalm/issues/2450#issuecomment-563479630",
```

Comment:
This ticket is closed, is this still relevant?

Reply:
b5e64dcfc148d2c8d968865aee4deebccf2aee35